src/auth.ts

```diff
@@ -105,15 +102,75 @@ export default class Auth extends AinftBase {
     appId: string,
     chain: string,
     network: string,
-    contractAddress: string,
+    contractAddress: string
   ): Promise<void> {
     const query = {
       appId,
       chain,
       network,
-      contractAddress
+      contractAddress,
     };
     const trailingUrl = `managedContracts`;
     return this.sendRequest(HttpMethod.DELETE, trailingUrl, query);
   }
+
+  async registerBlockchainApp(
+    appId: string,
+    userId: string,
+    accessAinAddress?: string
+  ) {
+    const ownerAddress = this.ain.wallet.defaultAccount?.address;
+    const body = {
+      appId,
+      userId,
+      ownerAddress,
+      accessAinAddress,
+    };
+    const trailingUrl = `register_blockchain_app`;
+    const { adminAddress, txHash } = await this.sendRequest(
+      HttpMethod.POST,
+      trailingUrl,
+      body
+    );
+
+    console.log(
+      `The gas fee for setting the admin account of app(${appId}) set in the nft server as the owner has been sent.`
```

Comment:
how about this?
The gas fee for setting the admin account as the owner of blockchain app has been sent.
It seems to be too long and confusing

Reply:
Done. Thanks!